app/helpers.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Parse a CSV field value as boolean.
+ *
+ * @param string $text
+ * @return boolean
+ */
+function str_to_boolean($text)
+{
+    $sanitized = strtolower($text);
+
+    if ($sanitized === 'y') {
+        return true;
+    }
+
+    return filter_var($sanitized, FILTER_VALIDATE_BOOLEAN);
```

Comment:
never knew about `FILTER_VALIDATE_BOOLEAN` - so cool! Definitely want to use and update some things in Rogue with this.